src/components/WwwFrame/TheHeader.vue

```diff
@@ -25,7 +25,7 @@
 				<kv-icon v-show="searchOpen" class="close-icon" name="x" />
 			</button>
 			<div class="flexible-center-area">
-				<div id="top-nav-search-area" :aria-hidden="searchOpen ? 'false' : 'true'">
+				<div id="top-nav-search-area" v-if="!hideSearchInHeader" :aria-hidden="searchOpen ? 'false' : 'true'">
```

Comment:
I know I approved, but should we use a v-show instead here... Just wondering if there are js events and bindings that may break if it's missing from the dom.

Reply:
@mcstover I did some testing toggling between /lend-by-category and /lend/filter and I searched through the code for references to the id `top-nav-search-area `. I feel we can keep this an if for now.